section3/exercises/loops-arrays.js

```diff
@@ -42,8 +58,13 @@ var names = ["Sevy", "Lindsey", "Jericho", "Raeonna", "Latrina", "Jhana", "Luca"
   // 18 or over? "Make sure you get out and vote!"
 
 // WORD FROM THE WISE: Before you start coding, jot down, with pen and paper, what steps you
-// want to tell the program to take. Think of all the concepts you've learned so far. 
+// want to tell the program to take. Think of all the concepts you've learned so far.
 
 // Remember, stuck for 15-20 minutes? Reach out to your Mod 0 cohort on Slack.
 
-var ages = [17, 20, 34, 18, 16, 65, 49, 10, 22, 18];
\ No newline at end of file
+var ages = [17, 20, 34, 18, 16, 65, 49, 10, 22, 18];
+
+for(var i = 0; i < ages.length; i++){
```

Comment:
Yep, nice solution! Most JS style guides won't allow for the single line if and else if, just a heads up!